WSL and Git Setup chapters (#168)
infinum/wordpress-handbook: Development environment setup/Git Setup.md

Git is a versioning software used to work on projects in a collaborative way. It's a developer's bread and butter. It acts as a safety net so that you can undo changes in a code you've written, but also as a documentation of the changes in your code.

We'll cover installing git on your system, and setting it up so that you can start working on the projects in no time.

## Installing git client

### Mac OS

First, try to see if it's already installed on your system

```bash
git --version
```

You may get prompted to install Xcode Command Line Tools. Once you have them installed, you should have git on your system.

> In case it wasn't installed, you can install it with Homebrew by running

Reviewer: ```suggestion
In case it isn't installed, you can install it with Homebrew by running
```

Reviewer: Yeah, if not changing the order as per the other two suggestions below, then this is good 🙂

Git is a versioning software used to work on projects in a collaborative way. It's a developer's bread and butter. It acts as a safety net so that you can undo changes in a code you've written, but also as a documentation of the changes in your code.

We'll cover installing git on your system, and setting it up so that you can start working on the projects in no time.

## Installing git client

### Mac OS

First, try to see if it's already installed on your system

```bash
git --version
```

You may get prompted to install Xcode Command Line Tools. Once you have them installed, you should have git on your system.

> In case it wasn't installed, you can install is by running

Reviewer: In case it wasn't installed, you can install it with Homebrew by running:

> In case it wasn't installed, you can install is by running

Reviewer: ```suggestion
In case it wasn't installed, you can install it with Homebrew by running
```

> 

Reviewer: ```suggestion
Just make sure you have [Homebrew](https://brew.sh/) set up before. 
```

```bash
HOMEBREW_NO_AUTO_UPDATE=1 brew install git
```

> Make sure you have [Homebrew](https://brew.sh/) set up before. 

Reviewer: I would move this to L:18

Reviewer: Sorry, will use "suggestion" in the future

> Make sure you have [Homebrew](https://brew.sh/) set up before. 

Reviewer: ```suggestion
```
